packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -336,7 +336,7 @@ export const BookerPlatformWrapper = (
     }))
   );
   const { data: overlayBusyDates } = useCalendarsBusyTimes({
-    loggedInUsersTz: session?.data?.timeZone || "Europe/London",
+    loggedInUsersTz: session?.data?.timeZone || timezone || "Europe/London",
```

Comment:
timezone comes from useTimePreference hook located in timePreferences.ts

Reply:
Shouldn't we also get rid of `session?.data?.timeZone` here ?

Reply:
indeed we should